docs/user-guide/remapping.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Small change. </p><p> </p><p>We can remap from the 480km grid to the 120km one, which would perform an upsampling operation. </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/807/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>